lib/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -68,7 +70,10 @@ const rule = (primary, _secondaryOptions, context) => {
 									return true;
 								}
 
-								complain(messages.expectedOperatorBeforeSign(operator), decl, operatorSourceIndex);
+								const endIndex = valueIndex + operatorSourceEndIndex;
+								const index = valueIndex + currentNode.sourceIndex;
+
+								complain(messages.expectedOperatorBeforeSign(operator), decl, index, endIndex);
```

Comment:
[ask] Honestly, I doubt if this change is valuable. The previous report seems natural to me. Any thoughts?

(Screenshots on VS Code ⬇️ )

### Before

<img width="896" alt="image" src="https://user-images.githubusercontent.com/473530/157663951-e04101e1-bc8c-457b-9ff7-04a39e139222.png">

### After

<img width="894" alt="image" src="https://user-images.githubusercontent.com/473530/157664084-871e2ed4-4d98-48d7-87a6-7668c1cc35aa.png">

Reply:
Before seems more natural. Let's revert.

Reply:
Thanks. I'll do it. 👍🏼

Reply:
Fixed via ec84751f